domain/credentials/service.go

```diff
@@ -90,6 +90,12 @@ func (s Service) GetCredentials(customer domain.Customer) ([]domain.Organization
 }
 
 func (s Service) SearchOrganizations(name, city string) ([]domain.OrganizationConceptCredential, error) {
+	if len(name) > 0 {
+		name += "*"
+	}
+	if len(city) > 0 {
+		city += "*"
+	}
```

Comment:
Does this break the search when the user already added a * to the end?
(And does that matter?)

Reply:
Yes, it matters (because then an actual * is required to match), but this way it doesn't change the current funcitonality.